proposals/README.md

```diff
@@ -58,8 +58,7 @@ Once the implementation phase is finished, we can include the contribution in th
 
 | ID  | Description  | Status  |
 |-----|--------------|---------|
-| [github issue] | adding this new encoding | POC |
-| [github issue] | add Variant type | Implementation |
+| [Issue #531](https://github.com/apache/parquet-format/issues/531) | FSST support for Parquet format | Implementation |
```

Comment:
I don't think we are at implementation yet.  I think we still want to work out:
1.  New page layout
2. Sharing dictionaries

The issue also doesn't have links to any of the docs produced so far?